chrome/content/zotero/elements/itemPane.js

```diff
@@ -492,7 +492,9 @@
 		getCurrentPane(mode = undefined) {
 			if (!mode) {
 				// Guess a mode from the current data
-				if (!this.data.length || this.data.length > 1) {
+				// No/multiple objects are selected OR selected object is a trashed collection/search
+				if (!this.data.length || this.data.length > 1
+					||this.data.every(obj => obj instanceof Zotero.Collection || obj instanceof Zotero.Search)) {
```

Comment:
This just needs to check `this.data[0]`, right? We're already excluding n != 1.

(And space after `||`)

Reply:
That's fair! Somehow `.every` felt more explicit to me - but it does only apply to `this.data[0]`, so I pushed that small tweak